app/containers/UserItem.tsx

```diff
@@ -64,14 +61,11 @@ const UserItem = ({ name, username, onPress, testID, onLongPress, style, icon, t
 		<View style={[styles.container, styles.button, style]}>
 			<Avatar text={username} size={30} style={styles.avatar} />
 			<View style={styles.textContainer}>
-				<Text style={[styles.name, { color: themes[theme].titleText }]} numberOfLines={1}>
+				<Text style={[styles.name, { color: themes[theme].bodyText }]} numberOfLines={1}>
 					{name}
 				</Text>
-				<Text style={[styles.username, { color: themes[theme].auxiliaryText }]} numberOfLines={1}>
-					@{username}
-				</Text>
 			</View>
-			{icon ? <CustomIcon name={icon} size={22} color={themes[theme].actionTintColor} style={styles.icon} /> : null}
+			{icon ? <CustomIcon name={icon} size={22} color={iconColor || themes[theme].actionTintColor} style={styles.icon} /> : null}
```

Comment:
This is possibly wrong and should have a unified color instead.
Let's discuss internally if we need this differentiation.